libs/moment-adapter/schematics/ng-add/schema.json

```diff
@@ -0,0 +1,27 @@
+{
+    "$schema": "http://json-schema.org/schema",
+    "id": "fundamental-ngx-ng-add",
+    "title": "Fundamental Library for Angular ng-add schematic",
+    "type": "object",
+    "properties": {
+        "animations": {
+            "type": "boolean",
+            "default": true,
+            "description": "Whether Angular browser animations should be set up.",
+            "x-prompt": "Set up browser animations for Fundamental Library for Angular?"
+        },
+        "styleFonts": {
```

Comment:
These properties are not used in the schematics script

Reply:
removed unnedded